pudb/run.py

```diff
@@ -4,34 +4,45 @@
 def main():
     import sys
 
-    from optparse import OptionParser
-    parser = OptionParser(
-            usage="usage: %prog [options] SCRIPT-TO-RUN [SCRIPT-ARGUMENTS]")
-
-    parser.add_option("-s", "--steal-output", action="store_true"),
-    parser.add_option("--pre-run", metavar="COMMAND",
+    import argparse
+    parser = argparse.ArgumentParser(
+        usage="%(prog)s [options] [-m] SCRIPT-OR-MODULE-TO-RUN [SCRIPT_ARGS]"
+    )
+    parser.add_argument("-s", "--steal-output", action="store_true"),
+    parser.add_argument("-m", "--module", action='store_true',
```

Comment:
Sure, I will add a comment!

However, I think it's best not to mention it in the help, I believe we should consider it as an implementation detail (i.e. people shouldn't rely that it should stay like this, and the fact that `-m -s --pre-run STUFF mymodule` works is just because "Simple is better than complex" and "practicality beats purity"). =)

Reply:
Done! Back to you :)

Reply:
Yeah this -m stuff should definitely be mentioned in a code comment. I don't know if the implementation details need to be outlined in the help. Stuff like `-m -s --pre-run STUFF mymodule` isn't actually supported. It just works because the only reasonable way to implement it with argparse requires it to be implemented that way.

Reply:
And I guess the comment in question is already there. So this looks fine to me (just from looking at it. I didn't test the latest code).